<launch>
    <!-- Load joint controller configurations from YAML file to parameter server -->
    <rosparam file="$(find uwrt_mars_rover_hw)/config/science_controllers.yaml" command="load"/>

    <group ns="/uwrt_mars_rover/">
        <node name="uwrt_mars_rover_hw_node" pkg="uwrt_mars_rover_hw" type="node"
              output="screen"/>
        <!-- load the controllers -->
        <node name="controller_spawner_test" pkg="controller_manager" type="spawner" respawn="false"
              output="screen" args="joint_state_controller
                                    indexer_state_controller
                                    elevator_joint_controller
                                    geneva_indexer_controller
                                    scooper_indexer_controller
                                    cover_indexer_controller"/>


        <!--    &lt;!&ndash; convert joint states to TF transforms for rviz, etc &ndash;&gt;-->
        <!--    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"-->
        <!--          respawn="false" output="screen">-->
        <!--        <remap from="/joint_states" to="/rrbot/joint_states"/>-->
        <!--    </node>-->
    </group>


</launch>
